src/Illuminate/Http/Middleware/TrustHosts.php

```diff
@@ -36,10 +37,10 @@ abstract public function hosts();
      * Handle the incoming request.
      *
      * @param  \Illuminate\Http\Request  $request
-     * @param  callable  $next
+     * @param  \Closure  $next
      * @return \Illuminate\Http\Response
      */
-    public function handle(Request $request, $next)
+    public function handle(Request $request, Closure $next)
```

Comment:
I spotted that the method signature didn't contain the `Closure` typehint here even though all other middleware do. Thought I'd fix this one up here as well while I'm at it.